spec/models/location_spec.rb

```diff
@@ -0,0 +1,92 @@
+require "rails_helper"
+
+RSpec.describe Location, type: :model do
+  describe "location model associations" do
+    let(:location) { create(:location) }
+
+    subject { location }
+
+    it "should have many events association" do
+      expect(subject.class.reflect_on_association(:events).macro).to eq(:has_many)
+    end
+
+    it "should have many races through events association" do
+      expect(subject.class.reflect_on_association(:races).macro).to eq(:has_many)
+    end
+
+    it "should have one cover_photo attached association" do
+      expect(subject.class.reflect_on_association(:cover_photo_attachment).macro).to eq(:has_one)
+    end
+
+    it "should have many promo_photos attached association" do
+      expect(subject.class.reflect_on_association(:promo_photos_attachments).macro).to eq(:has_many)
+    end
```

Comment:
This is wayyyyy overkill, but, probably a good thing to learn. Let's talk about where to draw the line on 'does this test make sense?'

Reply:
Cool. Yeah, I was down a model testing rabbit hole and unsure of where that line was.